### PR TITLE
added capillary docs

### LINKED ISSUE #5: Update partner price contracts
1. Two example - 1 with lat lan, one with zipcode
2. 3 errors response , if product not found response, radius is greater than max , zip code is not correct

### PR DESCRIPTION
**Pull request by caveadolfo22**

### CODE REVIEW

File: capillary-services/PARTNER-PRICE.md [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+        "universal_product_id_type": "ean",
+        "video": []
+    },
+    "item_type": "retailer_product"
+}
+```
+
+#### Possible errors
+
+##### Invalid uuid
```
Comment (yogi-bp) [inline capillary-services/PARTNER-PRICE.md]: uuid ? or universal_id ?

File: capillary-services/PARTNER-PRICE.md [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+
+
+#### Request
+
+```
+GET /capillary/best_price/partner_salespoints/get?radius=10&zip_code=06600 HTTP/1.1
+OR
+GET /capillary/best_price/partner_salespoints/get?radius=10&longitude=-99.163553&latitude=19.42586 HTTP/1.1
+X-api-key: byprice_own_key
+Host: 35.193.29.21:8005
```
Comment (yogi-bp) [inline capillary-services/PARTNER-PRICE.md]: Use api.salu., not IP

File: capillary-services/PARTNER-PRICE.md [hunk 3]
```diff
@@ -0,0 +1,441 @@
...
+- Description: Longitude of center point
+- Validation: Should be a proper longitude
+- Examples: -99.125781 or -99.112842
+
+**radius**
+
+- Data type: int, optional but strongly recommended
+- Description: Radius of area to get prices from
+- Validation: Should be an integer from 1 to 100
+- Examples: 10 or 30
```
Comment (yogi-bp) [inline capillary-services/PARTNER-PRICE.md]: Why 30 when you know our max limit is 10

File: capillary-services/PARTNER-PRICE.md [hunk 4]
```diff
@@ -0,0 +1,441 @@
...
+- Data type: int, optional but strongly recommended
+- Description: Zip code to get prices from
+- Validation: Should be an integer valid zip_code of 5 digits
+- Examples: 06600 or 04920
+
+
+#### Request
+
+```
+GET /capillary/website/item/get?uuid=6202a24c-c821-4f7a-bd96-f3b68bdef4c6&latitude=19.4326764&longitude=-99.1336339&radius=1 HTTP/1.1
```
Comment (yogi-bp) [inline capillary-services/PARTNER-PRICE.md]: This is not needed , it is only for internal use

**Review by yogi-bp — CHANGES_REQUESTED** [review]

File: capillary-services/PARTNER-PRICE.md (hunk 1 above)
Comment (caveadolfo22) [reply capillary-services/PARTNER-PRICE.md]: universal_id, done

File: capillary-services/PARTNER-PRICE.md (hunk 2 above)
Comment (caveadolfo22) [reply capillary-services/PARTNER-PRICE.md]: done

File: capillary-services/PARTNER-PRICE.md (hunk 3 above)
Comment (caveadolfo22) [reply capillary-services/PARTNER-PRICE.md]: changed

File: capillary-services/PARTNER-PRICE.md (hunk 4 above)
Comment (caveadolfo22) [reply capillary-services/PARTNER-PRICE.md]: removed

**Review by yogi-bp — APPROVED** [review]

**Pull request merged**